package.json

```diff
@@ -49,17 +49,18 @@
     "clean:build": "rimraf ./build/*",
     "clean:dist": "rimraf ./dist/*",
     "pretest": "npm run lint",
-    "test": "npm run clean:test && BABEL_ENV=coverage karma start karma.conf.js --single-run",
+    "test": "npm run clean:test && BABEL_ENV=coverage karma start karma.conf.js --single-run -- --max_old_space_size=4096",
+    "test:phantom": "npm run clean:test && BABEL_ENV=coverage karma start karma.conf.js --single-run --browsers PhantomJS -- --max_old_space_size=4096",
```

Comment:
Tha could probably be simplified to

    npm run test -- --browsers PhantomJS

Reply:
No i didn't work because we allready got `-- --max_old_space_size=4096` in `test`.